contracts/eden/include/_config.hpp.in

```diff
@@ -1,9 +1,16 @@
 #pragma once
 
 #include <eosio/asset.hpp>
+#include <eosio/name.hpp>
 
 namespace eden
 {
    inline constexpr eosio::symbol default_token{"${TOKEN_SYMBOL}", ${TOKEN_PRECISION}};
    inline constexpr uint64_t minimum_donation = ${TOKEN_MINIMUM_DONATION};
+
+   inline constexpr eosio::name atomic_assets_account = "${EDEN_ATOMIC_ASSETS_ACCOUNT}"_n;
+   inline constexpr eosio::name atomic_market_account = "${EDEN_ATOMIC_MARKET_ACCOUNT}"_n;
+   // TODO: These should be configured on init instead of being compiled into the contract
```

Comment:
should we also consider this for the default token and minimum donation?

Reply:
We should definitely *consider* it.  The default token will likely be the same for all instances on the same chain, and it should not be allowed to change post-genesis.

Reply:
On the other hand, I think it would be completely reasonable for different communities to have different minimum donations, and update the minimum donation at any time.  The minimum donation should be configurable.